timesketch/api/v1/resources.py

```diff
@@ -1373,12 +1373,14 @@ def post(self, sketch_id):
                 timeline = Timeline.query.get(timeline_id)
 
             # If enabled, run sketch analyzers when timeline is added.
-            if current_app.config.get(u'ENABLE_SKETCH_ANALYZERS'):
-                from timesketch.lib import tasks
-                pipeline = tasks.build_sketch_analysis_pipeline(
-                    sketch_id, searchindex_id)
-                if pipeline:
-                    pipeline.apply_async(task_id=searchindex_id)
+            # Import here to avoid circular imports.
+            from timesketch.lib import tasks
```

Comment:
so you removed both of the If sentences to check if current app config was set, so aren't you now always importing tasks? and doing that twice now, in line1049 and here

Reply:
Yep, I'm doing the check in tasks.build_sketch_analysis_pipeline instead, returning an empty list of analyzers if they are missing.